gui/src/components/logs/FormattedLogMessage.svelte

```diff
@@ -0,0 +1,21 @@
+<script lang="ts">
+  export let message: string = '';
+
+  let words = message.split(' ');
```

Comment:
This is going to create _tons_ of empty spans when there is whitespace in output (which is extremely common!).

A better approach would be to create a parse loop that searches for matching urls. To avoid retesting each element against the regex, you could create a small "AST". Something like `interface Word { link: boolean, value: string }`